src/builder-git.c

```diff
@@ -868,3 +868,32 @@ builder_git_checkout (const char     *repo_location,
 
   return TRUE;
 }
+
+char *
+builder_git_get_default_branch (const char *repo_location)
+{
+  char *default_branch;
+  char ** parts;
+  char ** parts2;
```

Comment:
```suggestion
  g_auto(GStrv) parts;
  g_auto(GStrv) parts2;
```

This should work I believe.